compiler/src/iree/compiler/Dialect/Flow/Transforms/DispatchWithTransformDialect.cpp

```diff
@@ -63,10 +94,41 @@ struct DispatchWithTransformDialect
     this->debugTransformRootTag = pass.debugTransformRootTag;
   }
 
+  void getPayloadRoots(SmallVectorImpl<Operation *> &targets) {
+    getOperation()->walk<WalkOrder::PreOrder>([&](linalg::LinalgOp linalgOp) {
+      targets.push_back(linalgOp);
+      return WalkResult::skip();
+    });
+  }
+
+  OwningOpRef<ModuleOp> constructTransformModule(MLIRContext *context) {
```

Comment:
Nit: pass loc explicitly and lift up the comment 1 level, no more need to pass context.